apps/dsaparam.c

```diff
@@ -25,12 +25,16 @@ NON_EMPTY_TRANSLATION_UNIT
 # include <openssl/x509.h>
 # include <openssl/pem.h>
 
+static int batch = 0;
```

Comment:
it is really necessary to add one more static global variable ?

Reply:
Just trying to be consistent with existing code.